docs/demos/filters/min_max_filter.jl

```diff
@@ -0,0 +1,56 @@
+# Topic : Max Min Filters
+# reference paper : https://core.ac.uk/download/pdf/194053536.pdf
```

Comment:
You can take a look at the existing demo on how to configure the meta data:

https://github.com/JuliaImages/ImageFiltering.jl/blob/5f4769ec0c8b0ec731aaa4c44ec6e6b33d1b0881/docs/demos/filters/median_filter.jl#L1-L6

I suggest we make a cover image for every demo we write; I don't like the default one (Although it's me that made the version...) Also refer to that example for how to generate a cover image (or gif).


![image](https://user-images.githubusercontent.com/8684355/95279729-03f59c80-0886-11eb-990e-232aad6dc53c.png)